### PR TITLE
change footer & navber file

### LINKED ISSUE #844: Update the blog link in the SEF Site
**Describe the bug**
Update the blog link in the entire website to the new link `https://sefglobal.medium.com/`
Please find the blog page access places in the following places
1) Navigation Bar
2) Footer

**Expected behavior**
The blog hyperlink should be redirected to the medium SEF Channel.

### PR DESCRIPTION
**Pull request by rocklikereeju00**
## Purpose

The purpose of this PR is to fix #844 

### Preview Link

https://pr-849-sef-site.surge.sh/

### CODE REVIEW

File: assets/content/static/navbar.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h6 class="text-primary mb-md-1">AcadeMix</h6>
                 </li>
 
                 <li class="nav-item dropdown">
-                    <a href="https://blog.sefglobal.org" class="nav-link" role="button">
+                    <a href="https://https://sefglobal.medium.com/" class="nav-link" role="button">
```
Comment (janethavi) [inline assets/content/static/navbar.html]: You have mistakenly added another `https://` to the link

**Review by janethavi — CHANGES_REQUESTED** [review]
The PR number is `850`. So the preview link should be like `https://pr-850-sef-site.surge.sh/`
If you can please add a small description about the changes you did to the `purpose` section too. You can edit this same PR. No need to close and send another one.

File: assets/content/static/navbar.html (hunk 1 above)
Comment (janethavi) [reply assets/content/static/navbar.html]: You can fix this issue in another commit in the same branch and push. So the PR will be updated automatically. No need to close the PR and send another one.

**Review by Gimhan-minion — COMMENTED** [review]
@rocklikereeju00 You can fix the changes @janethavi mentioned and we'll merge your PR. Please make sure you update the PR description. You can refer to our [closed PR's](https://github.com/sef-global/sef-site/pulls?q=is%3Apr+is%3Aclosed) to get an idea of how to add a detailed PR. If you have any doubts contact us via [SEF Hive](https://sef.discourse.group/). You can create a new thread and mention your issue there.

**Pull request closed without merging**